### PR TITLE
Ht20 39 implement new data in assets api

### PR DESCRIPTION
**Pull request by anefiox**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: Hackney.Shared.Asset/Domain/AssetCharacteristics.cs [hunk 1]
```diff
@@ -7,5 +9,17 @@ public class AssetCharacteristics
         public int NumberOfLivingRooms { get; set; }
         public string WindowType { get; set; }
         public string YearConstructed { get; set; }
+        public int NumberOfBedSpaces { get; set; }
+        public int NumberOfCots { get; set; }
+        public int ShowerRoomAmount { get; set; }
+        public int BathroomAmount { get; set; }
```
Comment (LBHMGeorgieva) [inline Hackney.Shared.Asset/Domain/AssetCharacteristics.cs:15]: Will that property be holding information about number of bathrooms? If yes, can it be renamed to 'NumberofBathrooms' for consistency?

File: Hackney.Shared.Asset/Domain/AssetCharacteristics.cs [hunk 2]
```diff
@@ -7,5 +9,17 @@ public class AssetCharacteristics
         public int NumberOfLivingRooms { get; set; }
         public string WindowType { get; set; }
         public string YearConstructed { get; set; }
+        public int NumberOfBedSpaces { get; set; }
+        public int NumberOfCots { get; set; }
+        public int ShowerRoomAmount { get; set; }
```
Comment (LBHMGeorgieva) [inline Hackney.Shared.Asset/Domain/AssetCharacteristics.cs:14]: Same question as below.

File: Hackney.Shared.Asset/Domain/AssetCharacteristics.cs [hunk 3]
```diff
@@ -7,5 +9,17 @@ public class AssetCharacteristics
...
+        public int BathroomAmount { get; set; }
+        public string AssetPropertyFolderLink { get; set; }
```
Comment (LBHMGeorgieva) [inline Hackney.Shared.Asset/Domain/AssetCharacteristics.cs:16]: What is the folder link? A Google drive folder location?

File: Hackney.Shared.Asset/Domain/AssetCharacteristics.cs (hunk 1 above)
Comment (anefiox) [reply Hackney.Shared.Asset/Domain/AssetCharacteristics.cs]: This has been changed

File: Hackney.Shared.Asset/Domain/AssetCharacteristics.cs (hunk 3 above)
Comment (anefiox) [reply Hackney.Shared.Asset/Domain/AssetCharacteristics.cs:16]: This is a link to a google drive folder

**Review by LBHMGeorgieva — APPROVED** [review]

**Pull request merged**